sky/templates/kubernetes-port-forward-proxy-command.yaml.j2

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+# Specifies which config file kubectl should use.
+export KUBECONFIG="{{ kube_config_path }}"
```

Comment:
Do we need this? We ideally want to operate with whatever `KUBECONFIG` var is set by the user. I know some parts of our code won't support custom `KUBECONFIG`, but we should avoid adding new code that puts strict dependency on a kubeconfig path set by us.

Reply:
Gotcha. Got them removed.